watershed-android/watershed/app/src/main/java/com/blueprint/watershed/Tasks/TaskFragment.java

```diff
@@ -201,4 +266,14 @@ private void setTasks(ArrayList<Task> tasks){
         public void onFragmentInteraction(Uri uri);
     }
 
+    public void showList() {
+        mNoTasks.setVisibility(View.GONE);
+        mListView.setVisibility(View.VISIBLE);
+    }
+
+    public void hideList() {
+        mNoTasks.setVisibility(View.VISIBLE);
```

Comment:
We should try to make use of the ListView.setEmptyView() rather than overlaying another on top of it.